### ISSUE TITLE
zoom, delete etc controls are not showing on any sticker

### ISSUE THREAD

**Issue by mubashar-javed1** [wuapnjie/StickerView#39]
Whenever i create new sticker and add it stickerview. current sticker do not show controls. i have put lot of effort. but all in vain.

**Comment by DastanIqbal** [comment]
try this
`stickerView.configDefaultIcons();`

`<com.xiaopo.flying.sticker.StickerView
            android:id="@+id/sticker_view"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:layout_gravity="center"
            android:background="#77ff0000"
            app:borderAlpha="123"
            app:borderColor="@android:color/white"
            app:bringToFrontCurrentSticker="true"
            app:showBorder="true"
            app:showIcons="true">`

**Comment by mubashar-javed1** [comment]
Thanks so much brother.. It's working 👍

**Comment by Ziaxp** [comment]
Great!
Can you please tell, how to hide the Anchor Icons when click outside? This happen only when we click on other object, then the BitmapIcons shifted to that object but I need to remove/hide it when click out of bound.

**Comment by pranishres** [comment]
Yes I also would like to access the `showBorder` and `showIcons`  attributes programatically

**Comment by Jack832** [comment]
@DastanIqbal Thanks i have spent almost 2 days to solve this.
